src/main/target/ALIENWHOOP/config.c

```diff
@@ -63,28 +67,34 @@
 #undef BRUSHED_MOTORS_PWM_RATE
 #endif
 
-#define BRUSHED_MOTORS_PWM_RATE 666           // 666Hz };-)>~ low PWM rate seems to give better power and cooler motors...
+#define BRUSHED_MOTORS_PWM_RATE 32000
 
 
 void targetConfiguration(void)
 {
     if (hardwareMotorType == MOTOR_BRUSHED) {
         motorConfigMutable()->dev.motorPwmRate = BRUSHED_MOTORS_PWM_RATE;
-        motorConfigMutable()->minthrottle = 1020;
+        motorConfigMutable()->minthrottle = 1050;
         motorConfigMutable()->maxthrottle = 2000;
     }
 
     /* Default to Spektrum */
-    rxConfigMutable()->serialrx_provider = SERIALRX_SPEKTRUM2048;
-    rxConfigMutable()->spektrum_sat_bind = 5;
+    rxConfigMutable()->serialrx_provider = SERIALRX_SPEKTRUM2048; // all DSM* except DSM2 22ms
+    rxConfigMutable()->spektrum_sat_bind = 5; // DSM2 11ms
     rxConfigMutable()->spektrum_sat_bind_autoreset = 1;
+    rxConfigMutable()->mincheck = 1025;
+    rxConfigMutable()->maxcheck = 2000;
+    rxConfigMutable()->rcInterpolation = RC_SMOOTHING_MANUAL;
+    rxConfigMutable()->rcInterpolationInterval = 14;
     parseRcChannels("TAER1234", rxConfigMutable());
 #if defined(ALIENWHOOPF7)
-    rxConfigMutable()->serialrx_inverted = true;
+    rxConfigMutable()->serialrx_inverted = false; // don't invert signal for target default DSM*
```

Comment:
This is already the default. Adding this in here will just create additional work for code maintenance, please drop. Also, please remove `maxthrottle` for the same reason.